apps/rush-lib/src/schemas/rush.schema.json

```diff
@@ -68,6 +68,10 @@
       "description": "If true, consistent version specifiers for dependencies will be enforced (i.e. \"rush check\" is run before some commands).",
       "type": "boolean"
     },
+    "normalizeChmodFieldInTarHeader": {
```

Comment:
@sachinjoseph  Why is this optional?  It feels like a bug fix.

If we want to introduce it conservatively (e.g. "opt-in" or "opt-out") in case it causes problems, you can put the setting in `experiments.json` instead.

It seems like a relatively safe change, though.

Reply:
@octogonz  - that was my intention - to make this an opt-in until we are more confident. I will add it to experiments.json